src/x86/macros.rs

```diff
@@ -350,3 +350,4109 @@ macro_rules! assert_approx_eq {
                  *a, *b, $eps, (*a - *b).abs());
     })
 }
+
+
+macro_rules! constify_bextri {
```

Comment:
So... What kind of hit does this have on compile times?

Reply:
Surprisingly low. The version using 65000 match arms took more than 2 hours to compile, and this version with 4000 match arms compiles just as fast as without the macro. There might be some quadratic or exponential behavior in rustc somewhere. I need to fill a bug.